chapters/hpmor-chapter-022.tex

```diff
@@ -505,7 +509,7 @@ \chapter{The Scientific Method}
 \item Magic itself is fading.
 \item Wizards are interbreeding with Muggles and Squibs.
 \item Knowledge to cast powerful spells is being lost.
-\item Wizards are eating the wrong foods as children, or something else besides blood is making them grow up weaker. \item Muggle technology is interfering with magic. (Since 800 years ago?) \item Stronger wizards are having fewer children. (Draco = only child? Check if 3  powerful wizards, Quirrell / Dumbledore / Dark Lord, had any children.) \end{enumerate} {\scshape Tests:} \end{center}
+\item Wizards are eating the wrong foods as children, or something else besides blood is making them grow up weaker. \item Muggle technology is interfering with magic. (Since 800 years ago?) \item Stronger wizards are having fewer children. (Draco = only child? Check if 3 powerful wizards, Quirrell / Dumbledore / Dark Lord, had any children.) \end{enumerate} {\scshape Tests:} \end{center}
```

Comment:
Only just noticed this: please can you unwrap the `\item`s on to separate lines?

Reply:
done